lib/util/selector-parser.js

```diff
@@ -0,0 +1,255 @@
+"use strict";
+
+//   selectors = selector [ws ',' ws selector]*;
```

Comment:
I think we will have to support it. Otherwise, we will be stuck explaining why people need to choose between using `:exit` and using other selectors.